CHANGELOG.md

```diff
@@ -1,11 +1,44 @@
 Changes by Version
 ==================
 
-2.19.1 (unreleased)
+2.20.0 (unreleased)
```

Comment:
According to RELEASE.MD: "Alter CHANGELOG.md from <placeholder_version> (unreleased) to <X.Y.Z> (YYYY-MM-DD)"